components/automate-ui/src/app/page-components/user-table/user-table.component.html

```diff
@@ -7,7 +7,9 @@
   <chef-toolbar>
     <app-authorized [allOf]="createPermissionsPath" [overrideVisible]="overridePermissionsCheck">
       <div *ngIf="showEmptyMessage || showTable" [ngClass]="showEmptyMessage ? 'empty-state' : ''">
-        <chef-button primary [disabled]="!addButtonEnabled" (click)="addClicked.emit($event)"> {{ addButtonText }}
+        <!-- TC: be careful when writing cypress tests that use data-cy binds in a re-useable component.
```

Comment:
I wonder if there's a more global place to put this warning. 🤔

Reply:
I'll add an entry to the readme.

Reply:
Have to step out. If it doesn't happen here I'll make a follow up PR.

Reply:
Done!